src/EthMultiVault.sol

```diff
@@ -135,7 +135,7 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
     BondingCurveConfig public bondingCurveConfig;
 
     /// @dev Gap for upgrade safety
-    uint256[48] private __gap;
```

Comment:
3.7 - Incorrectly Adjusted Gap Size in EthMultiVault

We adjusted the gap size correctly here to 47